src/test/java/org/junit/tests/experimental/theories/runner/UnsuccessfulWithDataPointFields.java

```diff
@@ -121,7 +121,7 @@ public void numbers(int x) {
     public void dataPointFieldsMustBeStatic() {
         assertThat(
                 testResult(DataPointFieldsMustBeStatic.class),
-                CoreMatchers.<PrintableResult>both(failureCountIs(2))
+                CoreMatchers.<PrintableResult>both(failureCountIs(1))
```

Comment:
Verifying the failure count does not make sense anymore. I think it should be removed. @junit-team/junit-committers What do you think?